sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -25,42 +25,27 @@ var tokenCredential = new ClientSecretCredential(
     clientSecret,
     new TokenCredentialOptions { AuthorityHost = KnownAuthorityHosts.AzureCloud });
 
-var dtClient = new DigitalTwinsClient(
+var client = new DigitalTwinsClient(
     new Uri(adtEndpoint),
     tokenCredential);
 ```
 
-Also, if you need to override pipeline behavior, such as provide your own HttpClient instance, you can do that via client options. This parameter is optional.
+Also, if you need to override pipeline behavior, such as provide your own HttpClient instance, you can do that via the other constructor that takes a client options.
+It provides an opportunity to override default behavior including:
 
-```C# Snippet:DigitalTwinsSampleCreateServiceClientInteractiveLogin
-// This illustrates how to specify client options, in this case, by providing an
-// instance of HttpClient for the digital twins client to use.
-var clientOptions = new DigitalTwinsClientOptions
-{
-    Transport = new HttpClientTransport(httpClient),
```

Comment:
I liked this example here.

Reply:
Me too. They said remove it from the readme, but I kept it for web documentation because we have nothing else interesting really to show for options. Maybe retry.